<launch>


  <arg name="map_file1"  default="$(find map_mux)/maps/atrium_with_elevators.yaml" />
  <arg name="map_file2"  default="$(find map_mux)/maps/atrium_with_elevators.yaml" />
  <arg name="map_file3"  default="$(find map_mux)/maps/map_whole2_with_elevators.yaml" />
  
  <node name="map_server1" pkg="map_server" type="map_server" args="$(arg map_file1)"> 
    <remap from="map" to="map1"/>
  </node>

  <node name="map_server2" pkg="map_server" type="map_server" args="$(arg map_file2)"> 
    <remap from="map" to="map2"/>
  </node>

  <node name="map_server3" pkg="map_server" type="map_server" args="$(arg map_file3)"> 
    <remap from="map" to="map3"/>
  </node>

  <node name="map_mux1" pkg="map_mux" type="main.py" /> 

</launch>
